check_msid_status.py

```diff
@@ -66,8 +45,14 @@ def check_status(msid, val, ldict, vdict):
                     'COSCS129S','COSCS130S', 'COSCS131S','COSCS132S','COSCS133S',\
                     'COSCS107S','CORADMEN', 'CCSDSTMF', 'ACAFCT', 'AOFSTAR',\
                     '2SHLDART', 'PLINE03T', 'PLINE04T', 'AACCCDPT', '3LDRTNO']:
-            status = check_status_letter(msid, val, vdict)
-            return status
+            try:
```

Comment:
General comment on check_status:
- Not sure if I like the logic behind try...except lines 32-36, we return there status "GREEN" in case of an error.
- lines 30-31+, try val = float(val), consider replacing this with
```
if isinstance(val, float) or isinstance(val, int):
    ...
elif val in ['NaN, 'nan', '']:
    ...
elif msid in []:
    ...
elif val in ['ERR', ...]:
    ...
else:
    return 'GREEN'
```
- why try...except is needed for check_status_letter?
- correct typos (nemeric, neumeric -> numeric)
- I think we should be returning a neutral status, e.g., "WHITE" for the final unaccounted for case rather than "else return 'GREEN'"
- in check_status_neumeric(), line 77, replace try...except, and limit = ldict[msid] with the condition ``if msid in ldict.keys():``
- in check_status_neumeric(), conditions in line 82 do not seem to be needed, ``if (len(limit) == 0) or (limit == ['']):`` because the CSH_limit_table.json file should not contain invalid limits
- in check_status_neumeric(), try...except starting in line 88 should not bee needed because the type of the value was already checked in check_state